<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TlgEditorWindow</class>
 <widget class="QMainWindow" name="TlgEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор телеграмм</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Телеграмма</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QToolButton" name="codecBtn">
          <property name="text">
           <string>UTF-8</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>702</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="TlgEditor" name="tlgEditor"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="decodeTab">
       <attribute name="title">
        <string>Сводки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="closeBtn">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Телеграмма</string>
    </property>
    <addaction name="newAct"/>
    <addaction name="newWmoCircAct"/>
    <addaction name="newHmsCircAct"/>
    <addaction name="newWmoAddrAct"/>
    <addaction name="newHmsAddrAct"/>
    <addaction name="openAct"/>
    <addaction name="saveAct"/>
    <addaction name="saveAsAct"/>
    <addaction name="separator"/>
    <addaction name="sendAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Вставить символ</string>
     </property>
     <addaction name="insertSohAct"/>
     <addaction name="insertCrAct"/>
     <addaction name="insertLfAct"/>
     <addaction name="insertStxAct"/>
     <addaction name="insertEtxAct"/>
     <addaction name="insertSoAct"/>
     <addaction name="insertSiAct"/>
    </widget>
    <addaction name="menu_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="insertSohAct"/>
   <addaction name="insertStxAct"/>
   <addaction name="insertEtxAct"/>
   <addaction name="insertSoAct"/>
   <addaction name="insertSiAct"/>
   <addaction name="insertCrAct"/>
   <addaction name="insertLfAct"/>
  </widget>
  <action name="openAct">
   <property name="icon">
    <iconset resource="../../../etc/novost.icons.qrc">
     <normaloff>:/novost/icons/file_open.png</normaloff>:/novost/icons/file_open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="icon">
    <iconset resource="../../../etc/novost.icons.qrc">
     <normaloff>:/novost/icons/save.png</normaloff>:/novost/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="icon">
    <iconset resource="../../../etc/novost.icons.qrc">
     <normaloff>:/novost/icons/close.png</normaloff>:/novost/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="newAct">
   <property name="text">
    <string>Новая [пустая]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="insertCrAct">
   <property name="text">
    <string>[&amp;CR]</string>
   </property>
  </action>
  <action name="insertSohAct">
   <property name="text">
    <string>[&amp;SOH]</string>
   </property>
  </action>
  <action name="insertLfAct">
   <property name="text">
    <string>[&amp;LF]</string>
   </property>
  </action>
  <action name="insertStxAct">
   <property name="text">
    <string>[S&amp;TX]</string>
   </property>
  </action>
  <action name="insertEtxAct">
   <property name="text">
    <string>[&amp;ETX]</string>
   </property>
  </action>
  <action name="insertSoAct">
   <property name="text">
    <string>[S&amp;O]</string>
   </property>
  </action>
  <action name="insertSiAct">
   <property name="text">
    <string>[S&amp;I]</string>
   </property>
  </action>
  <action name="saveAsAct">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="sendAct">
   <property name="text">
    <string>Отправить в ЦКС...</string>
   </property>
  </action>
  <action name="stormAct">
   <property name="text">
    <string>Разослать по &quot;штормовому кольцу&quot;</string>
   </property>
  </action>
  <action name="newWmoCircAct">
   <property name="text">
    <string>Новая [циркулярная, ВМО]</string>
   </property>
  </action>
  <action name="newWmoAddrAct">
   <property name="text">
    <string>Новая [адресная, ВМО]</string>
   </property>
  </action>
  <action name="newHmsCircAct">
   <property name="text">
    <string>Новая [циркулярная, ГМС]</string>
   </property>
  </action>
  <action name="newHmsAddrAct">
   <property name="text">
    <string>Новая [адресная, ГМС]</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TlgEditor</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/tlgeditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>closeBtn</tabstop>
  <tabstop>codecBtn</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../../etc/novost.icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitAct</sender>
   <signal>triggered()</signal>
   <receiver>TlgEditorWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>TlgEditorWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
